### PR TITLE
Kolban 2015 12 17

### PR DESCRIPTION
**Pull request by nkolban**
Test of our pull request and 1st pass at some Makefiles to create a build.

### CODE REVIEW

File: .cproject [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (me-no-dev) [inline .cproject]: put the files that eclipse generates in .gitignore please :) they are platform specific for most part and not really in the scope.

File: piduino/core/Arduino.c [hunk 2]
```diff
@@ -390,6 +390,7 @@ static int map_registers(uint32_t reg_offset){
 void init_pins(){
   uint32_t revision = getBoardRev();
   if(revision == 0xa01041) revision = 0x10;//pi2 has B+ pinout
+  if(revision == 0xa21041) revision = 0x13;//pi2 has B+ pinout
```
Comment (me-no-dev) [inline piduino/core/Arduino.c]: did this work for you? interesting why I had different number before? will check my PI2 and see if it's the same.

File: .cproject (hunk 1 above)
Comment (nkolban) [inline .cproject]: Done and did.  The Eclipse files are now in .gitignore.  I think we will have to add them as part of this Pull Request and then in a future commit remove them.

Comment (me-no-dev) [inline .cproject]: why are they still visible though?

Comment (nkolban) [inline .cproject]: I'm not too experienced in working with Git outside the norm.  When I checked them in, they got added to the pull-request.  You then asked for them to be removed from the distribution ... so I added them to gitignore.  However, this means that they are still in the pull request.  The only way I know to remove them would be for the pull-request to be completed, you to sync your clone, remove them in your clone and then you commit your project with the files removed.  My environment will no longer attempt to try and include them in future commits.

**Pull request merged**